apics/static/Atlas/128.html

```diff
@@ -1,14 +1,3 @@
-<body><p class="c10">While all <i>APiCS</i>
-languages have the widespread nasal consonants [m] (bilabial nasal)
-and [n] (dental/alveolar nasal), there is much more variation in
-the occurrence of the palatal nasal [ɲ], i.e. the sound of Spanish
-<i>señor,</i> Portuguese <i>senhor</i> and French <i>seigneur</i>
-(‘lord, sir’).</p><p>value-table</p><p class="c10">One would expect that
```

Comment:
Note that `<p>value-table</p>` marks the place where the feature value table will be dynamically inserted. Is it intentional to remove this table as well?

Reply:
@DarjaAppelganz any comment on this?

Reply:
no, it was not - thank you a lot